packages/leva/src/components/Leva/Filter.tsx

```diff
@@ -74,30 +92,37 @@ export function TitleWithFilter({ setFilter, onDrag, toggle, toggled }: TitleWit
 
   return (
     <>
-      <StyledTitleWithFilter>
+      <StyledTitleWithFilter mode={drag ? 'drag' : undefined}>
         <Icon active={!toggled} onClick={() => toggle()}>
           <Chevron toggled={toggled} width={12} height={8} />
         </Icon>
-        <Drag {...bind()}>
-          <svg width="20" height="10" viewBox="0 0 28 14" xmlns="http://www.w3.org/2000/svg">
-            <circle cx="2" cy="2" r="2" />
-            <circle cx="14" cy="2" r="2" />
-            <circle cx="26" cy="2" r="2" />
-            <circle cx="2" cy="12" r="2" />
-            <circle cx="14" cy="12" r="2" />
-            <circle cx="26" cy="12" r="2" />
-          </svg>
-        </Drag>
-        <Icon active={filterShown} onClick={() => setShowFilter((f) => !f)}>
-          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 0 20 20">
-            <path d="M9 9a2 2 0 114 0 2 2 0 01-4 0z" />
-            <path
-              fillRule="evenodd"
-              d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-13a4 4 0 00-3.446 6.032l-2.261 2.26a1 1 0 101.414 1.415l2.261-2.261A4 4 0 1011 5z"
-              clipRule="evenodd"
-            />
-          </svg>
-        </Icon>
+        <TitleContainer {...(drag ? bind() : {})} mode={drag ? 'drag' : undefined}>
+          {title === undefined && drag ? (
```

Comment:
Since the dots indicate drag behaviour it seems to make sense to now show them when dragging is disabled.